src/Jaeger.Core/SpanBuilder.cs

```diff
@@ -247,7 +247,7 @@ private SpanContext CreateChildContext()
 
             return new SpanContext(
                 preferredReference.TraceId,
-                SpanId.NewUniqueId(),
+                SpanId.NewUniqueId(_tracer),
```

Comment:
this may result in 128bit span ID, which are not supported.

Reply:
While both SpanId and TraceId now take a Tracer as context, only TraceId has the code to optionally create 128 bit spans. This pre-existing test proves that:

https://github.com/jaegertracing/jaeger-client-csharp/blob/5fe4c9becb4653262ed41ecc151b264533852ca1/test/Jaeger.Core.Tests/SpanTests.cs#L54